recipes/cppcheck/all/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include("conanbuildinfo.cmake")
+conan_basic_setup(TARGETS)
```

```suggestion
include(conan_paths.cmake)
conan_basic_setup(TARGETS)
```
so that you don't have to `set(PCRE_LIBRARY CONAN_PKG::pcre)`